draft-ietf-protected-audience-key-value-services.md

```diff
@@ -579,9 +613,51 @@ The output is a `response` to be sent to a Client.
    `rctxt`, as described in {{response-encryption}}. Return an empty `response` on failure.
 9.  Return `response`.
 
-### Parsing a Response
-
-TODO
+### Parsing a Response (#response-parsing)
+
+This section describes how a conforming Client MUST parse and validate a response from a Trusted Key Value service. 
+
+It takes as input the `request context` returned from {{request-generate}} in addition to the `encrypted response`.
+
+The output is a `result` map, where the keys are strings, and the values are maps with both keys and values as strings.
+
+1. Use `request context` as the context to decrypt `encrypted response` and obtain `framed response`, returning failure if decryption fails.
+1. Remove and extract the first 5 bytes from `framed response` as the framing header (described in {{framing}}), removing them from `framed response`.
+1. If `framing header`'s `Version` field is not 0 or 2, return failure.
```

Comment:
The framing header doesn't have a Version field. Do you mean "Compression"?

Reply:
Yeah, changed it to `Compression`.